grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -91,6 +92,25 @@ public GrpcServiceBuilder addService(BindableService bindableService) {
         return addService(bindableService.bindService());
     }
 
+    /**
+     * Adds a gRPC {@link BindableService}s to this {@link GrpcServiceBuilder}. Most gRPC service
+     * implementations are {@link BindableService}s.
+     */
+    public GrpcServiceBuilder addServices(BindableService... bindableServices) {
+        requireNonNull(bindableServices, "bindableServices");
+        return addServices(ImmutableList.copyOf(bindableServices));
+    }
+
+    /**
+     * Adds a gRPC {@link BindableService}s to this {@link GrpcServiceBuilder}. Most gRPC service
+     * implementations are {@link BindableService}s.
+     */
+    public GrpcServiceBuilder addServices(Iterable<BindableService> bindableServices) {
+        requireNonNull(bindableServices, "bindableServices");
+        bindableServices.forEach(s -> addService(s.bindService()));
```

Comment:
How about `bindableServices.forEach(this::addService)`?

Reply:
It would be better! Fixed. :-)